src/js/utils/define.js

```diff
@@ -0,0 +1,8 @@
+export const duplicationOperator = "duplication operator";
+export const maxLength = "max length";
+export const plus = "+";
+export const minus = "-";
+export const division = "/";
+export const multiplication = "X";
+export const equal = "=";
```

Comment:
```suggestion
export const Operator = {
  PLUS: "+",
  MINUS: "-",
  MULTIPLE: "X",
  DIVISION: "/",
  EQUALS: "=",
}
```
보통 이렇게 묶어서 상수로 표현한답니다!